docs/user/user/adding-glean-to-your-project/enable-glean-dictionary.md

```diff
@@ -0,0 +1,41 @@
+# Adding your product to the Glean Dictionary
+
+This page provides a step-by-step guide on how to add your project to the [Glean Dictionary](https://dictionary.telemetry.mozilla.org/).
+
+This is just one of the required steps for integrating Glean successfully into a project. Check the full [Glean integration checklist](./index.md) for a comprehensive list of all the steps involved in doing so.
+
+# Requirements
+
+- Circle CI
+
+# Enable your product
+
+At least one week before releasing your product, [file a data engineering bug][dataeng-bug] to
+enable your product's application id and metrics to be published to the Glean Dictionary.
+
+# Validate and publish metrics
+
+After your product has been enabled, you must submit commits to [`probe-scraper`](https://github.com/mozilla/probe-scraper) to validate and publish metrics to the Glean Dictionary. Metrics will only be published from branches defined in [`probe-scraper/repositories.yaml`](https://github.com/mozilla/probe-scraper/blob/main/repositories.yaml) when your product was enabled, or the git default branch if not explicitly configured.
+
+Use the `probe-scraper-glean-push` docker image in your `.circleci/config.yml` to submit commits:
+
+```yaml
+---
+version: 2.1
+
+jobs:
+  glean-push:
+    docker:
+      - gcr.io/relud1/probe-scraper-glean-push:latest
```

Comment:
TODO: replace `relud1` with the GCP project provided in [DSRE-684](https://mozilla-hub.atlassian.net/browse/DSRE-684)